src/utils/gql/queries.ts

```diff
@@ -0,0 +1,23 @@
+import gql from "graphql-tag";
```

Comment:
queries are not a util. putting the gql code in the utils folder is something i have felt is wrong for a while now. maybe this is the time to move it to a different directory.